src/android/jni/sensor.cpp

```diff
@@ -8,6 +8,55 @@
 
 #include "../../../include/librealsense2/rs.h"
 
+#include "jni_logging.h"
+#include "jni_user.h"
+
+extern "C"
+JNIEXPORT void JNICALL
+Java_com_intel_realsense_librealsense_Sensor_nOpen(JNIEnv *env, jclass type, jlong handle, jlong sp) {
+    rs2_error* e = nullptr;
+    rs2_open(reinterpret_cast<rs2_sensor *>(handle), reinterpret_cast<rs2_stream_profile *>(sp), &e);
+    handle_error(env, e);
+}
+
+static rs_jni_userdata sdata = {NULL, 0, JNI_FALSE, NULL, NULL};
+
+extern "C"
+JNIEXPORT void JNICALL
+Java_com_intel_realsense_librealsense_Sensor_nStart(JNIEnv *env, jclass type, jlong handle, jobject jcb) {
+    rs2_error* e = nullptr;
+
+    if (rs_jni_callback_init(env, jcb, &sdata) != true) return;
+
+    auto cb = [&](rs2::frame f) {
+        rs2_error* e = nullptr;
+
+        rs_jni_cb(f, &sdata);
+        return NULL;
```

Comment:
can you explain the return NULL here, please?

Reply:
Removed.